packages/utils/vue/install.ts

```diff
@@ -30,7 +30,10 @@ export const withInstallFunction = <T>(fn: T, name: string) => {
   return fn as SFCInstallWithContext<T>
 }
 
-export const withInstallDirective = <T>(directive: T, name: string) => {
+export const withInstallDirective = <T extends Directive<any, any>>(
```

Comment:
We can simplify it like this:

```suggestion
export const withInstallDirective = <T extends Directive>(
```

Reply:
Yep!